charts/site-stack/Chart.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v0
```

Comment:
Not sure this is the one you want... apiVersion: v2 is standard

Reply:
ok I will change it right now